README.md

```
@@ -150,10 +152,11 @@ Name | Description
 
 Name | Type | Description
 --- | --- | ---
-[redhatci.ocp.junit2dict]() | Filter | Transforms a JUnit into a dictionary
-[redhatci.ocp.junit2obj]() | Filter | Transforms a JUnit XML into a corresponding JSON text
+[redhatci.ocp.junit2dict]() | Filter | Converts a JUnit XML report passed **by file name** into a list of dictionaries, i.e. JSON
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)